### PR TITLE
chore(i18n): add russian translation

### PR DESCRIPTION
**Pull request by riichi67**

### CODE REVIEW

File: client/src/locale/ru-RU.locale.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                    'Pacstall использует стабильную базу Ubuntu, но позволяет использовать ' +
+                    "bleeding-edge программы почти что без компромиссов, так что вам не " +
+                    'нужно беспокоиться о патчах безопасности или новом функционале.',
+            },
+            howItWorks: {
+                title: 'Тогда как это работает?',
+                description:
+                    ' Pacstall takes in files known as <0>pacscripts</0> (similar' +
+                    ' to PKGBUILDs) that contain the necessary contents to build packages,' +
+                    ' and builds them into executables on your system.',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:20]: Why can't these be translated?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: client/src/locale/ru-RU.locale.ts (hunk 1 above)
Comment (riichi67) [reply client/src/locale/ru-RU.locale.ts:20]: my bad, forgot about those, will fix in a bit!

Comment (riichi67) [reply client/src/locale/ru-RU.locale.ts]: fixed now!

File: client/src/locale/ru-RU.locale.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
+import Locale from './locale'
+
+export default <Locale>{
+    home: {
+        title: 'Pacstall',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:5]: The title should be translated also.

File: client/src/locale/ru-RU.locale.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                description:
+                    ' Pacstall использует файлы, известные как <0>pacscript-ы</0> (похоже' +
+                    ' на PKGBUILD-ы) которые содержат обязательные компоненты для сборки пакетов,' +
+                    ' и собирает из них исполняемые файлы на вашей системе.',
+            },
+        },
+        installationInstructions: 'Инструкция по установке',
+        showcase: {
+            title: 'Showcase',
+            packageSearch: 'Package Search',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:26]: need to be translated.

File: client/src/locale/ru-RU.locale.ts [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+            helpTranslate: 'Помочь с переводом',
+            becomeAMaintainer: 'Станьте сопровождающим пакета',
+        },
+        social: {
+            title: 'Соцсети',
+            discord: 'Discord',
+            matrix: 'Matrix',
+            reddit: 'Reddit',
+            lemmy: 'Lemmy',
+            mastodon: 'Mastodon',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:43]: Here.

File: client/src/locale/ru-RU.locale.ts [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+        },
+        browse: {
+            title: 'Browse Packages',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:46]: Here.

File: client/src/locale/ru-RU.locale.ts [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+            'Продолжая пользоваться этим сайтом, вы <strong>соглашаетесь с политикой конфиденциальности</strong>.',
+        ],
+        accept: 'Окей',
+    },
+    packageSearch: {
+        dropdown: {
+            package: 'Package',
+            packageTooltip: 'Searches in package names and descriptions',
+            maintainer: 'Maintainer',
+            maintainerTooltip: 'Searches by maintainer names and emails',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:68]: Here.

File: client/src/locale/ru-RU.locale.ts [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+        versionTooltip: {
+            notInRegistry: 'Этот пакет не в реестре Repology',
+            latest: 'Этот пакет последней версии',
+            patch: 'Для этого пакета есть доступный патч',
+            minor: 'Для этого пакета есть незначительное обновление',
+            major: 'Для этого пакета есть крупное обновление',
+            isGit: 'Этот пакет - это Git-пакет',
+        },
+        noResults: 'Не находите то, что хотели? <0>Создайте запрос!</0>',
+        search: 'Search',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:85]: Here.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: client/src/locale/ru-RU.locale.ts (hunk 4 above)
Comment (riichi67) [reply client/src/locale/ru-RU.locale.ts:43]: same goes for these, i can transliterate these, but these are names, and they're usually left as original ones in english

**Comment by Elsie19** [comment]
Yes please transliterate the name.

-------- Original Message --------

File: client/src/locale/ru-RU.locale.ts (hunk 4 above)
Comment (Elsie19) [reply client/src/locale/ru-RU.locale.ts:43]: Ok then keep those specifically.

**Comment by riichi67** [comment]
> Yes please transliterate the name.

i transliterated every name, hope everything's good now!

File: client/src/locale/ru-RU.locale.ts [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+export default <Locale>{
+    home: {
+        title: 'Паксталл',
+        subtitle: 'AUR для Ubuntu',
+        cards: {
+            whyDifferent: {
+                title: 'Чем это отличается от любого другого пакетного менеджера?',
+                description:
+                    'Паксталл использует стабильную базу Ubuntu, но позволяет использовать ' +
+                    "bleeding-edge программы почти что без компромиссов, так что вам не " +
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:12]: Is there an idiom for bleeding-edge you could use?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: client/src/locale/ru-RU.locale.ts (hunk 8 above)
Comment (Elsie19) [reply client/src/locale/ru-RU.locale.ts:12]: Any updates?

Comment (riichi67) [reply client/src/locale/ru-RU.locale.ts]: sorry for late response, fixed it!

File: client/src/locale/ru-RU.locale.ts [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+            pacstallDependencies: 'Зависимости Паксталл',
+            name: 'Название',
+            close: 'Закрыть',
+            provider: 'Провайдер',
+            noDescription: 'Нет доступного описания',
+            version: 'Версия',
+        },
+        requiredByModal: {
+            title: 'Требуется',
+            name: 'Name',
```
Comment (Elsie19) [inline client/src/locale/ru-RU.locale.ts:128]: Last one 😭

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: client/src/locale/ru-RU.locale.ts (hunk 9 above)
Comment (riichi67) [reply client/src/locale/ru-RU.locale.ts]: done!

**Review by Elsie19 — APPROVED** [review]
Thanks for contributing your language and time! Just to let you know, Russian is the second most used language on our site and until now we didn't have a translation for it, so we thank you so much! You're gonna help so many people.

**Comment by Elsie19** [comment]
The website is up and running with the Russian translations!

**Pull request merged**